UnityProject/Assets/Scripts/Player/Pettable.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+/// <summary>
+/// Allows an object to be pet by a player. Shameless copy of Huggable.cs
+/// </summary>
+public class Pettable : MonoBehaviour, IClientInteractable<PositionalHandApply>
+{
+	public bool Interact(PositionalHandApply interaction)
+	{
+		var targetNPCHealth = interaction.TargetObject.GetComponent<LivingHealthBehaviour>();
+		var performerPlayerHealth = interaction.Performer.GetComponent<PlayerHealth>();
+		var performerRegisterPlayer = interaction.Performer.GetComponent<RegisterPlayer>();
+
+		// Is the target in range for a pet? Is the target conscious for the pet? Is the performer's intent set to help?
+		// Is the performer's hand empty? Is the performer not stunned/downed? Is the performer conscious to perform the interaction?
+		// Is the performer interacting with itself?
+		if (!PlayerManager.LocalPlayerScript.IsInReach(interaction.WorldPositionTarget, false) ||
```

Comment:
Why not use `Validations.CanApply` instead? it has all default validations in place

Reply:
Its basically a copy paste of huggable.cs. Went ahead and converted it over to `Validations.CanApply`.

Reply:
Not 100% regarding the NetworkSide so if that needs adjusted just give me a heads up.